aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -37,10 +37,44 @@ if [[ -z $TOKEN ]]; then
 	exit 1
 fi
 
+# Add the secomp profile if it doesnt exist
+read -r -d '' _SECCOMP_PROFILE <<'EOF'
+{
+	"defaultAction": "SCMP_ACT_ERRNO",
+	"syscalls": [{
+		"names": ["capget", "capset", "chdir", "fchown", "futex", "getdents64", "getpid", "getppid", "lstat", "openat", "prctl", "setgid", "setgroups", "setuid", "stat", "rt_sigaction", "mprotect", "brk", "close", "open", "write", "mmap", "rt_sigprocmask", "sched_getaffinity", "arch_prctl", "access", "getrandom", "sigaltstack", "getrlimit", "set_tid_address", "fstat", "stat", "setsockopt", "read", "openat", "clone", "set_robust_list", "ioctl", "execve", "gettid", "socket", "munmap", "futex", "bind"],
+		"action": "SCMP_ACT_ALLOW",
+		"args": [],
+		"comment": "Necessary syscalls for working of container",
+		"includes": {},
+		"excludes": {}
+	}]
+}
+EOF
+
+read -r -d "" _POST_VAR <<EOF
+{
+   "name":"seccomp-default",
+   "type":"security.profile",
+   "description":"Ethos Default RunTime Profile",
+   "encrypt_all_envs":true,
+   "seccomp_profile":`echo $_SECCOMP_PROFILE | jq '@json'`,
+   "type":"security.profile",
+   "version":"1.0"
+}
+EOF
+
+EXISTING_SECOMP_RULE=$(curl --write-out %{http_code} --silent --output /dev/null -H "$HEADER: Bearer $TOKEN" $WEB_URL/securityprofiles/seccomp-default)
+if [[ "$EXISTING_SECOMP_RULE" == "200" ]]; then
+        log "seccomp-default exists..."
+else
+        curl --silent -H "Content-Type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d "$_POST_VAR" $WEB_URL/securityprofiles
+fi
+
 # See if profile already exists
-EXISTING_RULE=$(curl --write-out %{http_code} --silent --output /dev/null -H "$HEADER: Bearer $TOKEN" $WEB_URL/adminrules/core-user-rule)
+EXISTING_SECOMP_RULE=$(curl --write-out %{http_code} --silent --output /dev/null -H "$HEADER: Bearer $TOKEN" $WEB_URL/adminrules/core-user-rule)
```

Comment:
@kravivar - why are you overwriting the existing rule `EXISTING_RULE` with `EXISTING_SECOMP_RULE` when `EXISTING_SECOMP_RULE` already exists above (see line 67)?